apps/server/test/lexical/convertibles/lexical.plugin.diagnostic.result_test.exs

```diff
@@ -73,7 +73,7 @@ defmodule Lexical.Convertibles.Lexical.Plugin.V1.Diagnostic.ResultTest do
     test "it can translate a diagnostic that starts after an emoji", %{uri: uri} do
       assert {:ok, %Types.Diagnostic{} = converted} = to_lsp(plugin_diagnostic(uri, {6, 10}), uri)
 
-      assert converted.range == range(:lsp, position(:lsp, 5, 7), position(:lsp, 6, 0))
+      assert converted.range == range(:lsp, position(:lsp, 5, 10), position(:lsp, 6, 0))
```

Comment:
It makes sense that the converted UTF16 code unit should be 10 because the input UTF8 position was 10 and there's a 2-code-unit character, so you get 10 - 1 (account for 0-based) + 1 (account for extra code unit in the emoji).